docs/application/dotnet/guides/user-interface/nui/nui-components/AlertDialog.md

```diff
@@ -0,0 +1,34 @@
+# AlertDialog
+
+AlertDialog class shows a dialog with title, message, and action buttons.
+
+![AlertDialog](../../screenshots/AlertDialog.png)

```

Comment:
Global.
tizen-docs repository is using media folder in each directory, to hold the images to be inserted. Also, the file name to import an image is case-sensitive.

Actual file name : ../AlertDialog.png
Used file name : ./AlertDialog.PNG

So the image is broken.

Please, 
1. use "nui/nui-components/media" folder to handle the images, and 
2. check the capital letters in the file name.

Check other pages also to fix the broken images.